css/style.css

```diff
@@ -432,6 +432,15 @@ video {
 	opacity: .7;
 }
 
+.nameIndicator button.video-available {
```

Comment:
The class is added, if no video track is in the stream, right? If so, the name of the class should not be `video-available` but something like `no-video-tracks` / `no-video-available` (or similar).

Reply:
no-video-tracks sounds good to me